cmd/d2/help.go

```diff
@@ -15,9 +15,9 @@ import (
 
 func help(ms *xmain.State) {
 	fmt.Fprintf(ms.Stdout, `Usage:
-  %s [--watch=false] [--theme=0] file.d2 [file.svg]
+  %s [--watch=false] [--theme=0] file.d2 [file.svg|file.png]
 
-%[1]s compiles and renders file.d2 to file.svg
+%[1]s compiles and renders file.d2 to svg or png.
```

Comment:
.

Reply:
Not sure what this means

Reply:
Yeah but i'm not sure which comment hes asking to repeat. is it the "|" one or the "," one

Reply:
let's do `|`